maintainers/github-teams.json

```diff
@@ -61,7 +60,6 @@
       "happysalada": 5317234
     },
     "members": {
-      "Br1ght0ne": 12615679,
```

Comment:
As the maintainer for the beam team, @happysalada, please implement the removal of `Br1ght0ne` from the team.

Reply:
Now done myself in https://github.com/NixOS/nixpkgs/pull/471116